solutions/devsprint-riccardo-1/OnboardingChallenge/Screens/List/Cell/RepositoryCellView.swift

```diff
@@ -0,0 +1,91 @@
+//
+//  RepositoryCellView.swift
+//  OnboardingChallenge
+//
+//  Created by Mateus Nazário Coelho on 16/12/21.
+//
+
+import UIKit
+
+class RepositoryCellView: UITableViewCell {
+    // MARK: Outlets
+    private lazy var labelRepositoryName: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.boldSystemFont(ofSize: 18)
+        label.textColor = .black
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    private lazy var labelRepositoryOwnerName: UILabel = {
+        let label = UILabel()
+        label.font = UIFont.systemFont(ofSize: 14)
+        label.textColor = .gray
+        label.translatesAutoresizingMaskIntoConstraints = false
+        return label
+    }()
+    
+    // MARK: Initialization
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+        setupViews()
+    }
+    
+    required init?(coder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+    
+    override func awakeFromNib() {
```

Comment:
I removed, but didn't understand why I should remove the awakeFromNib ahaha can you explain me please?

Reply:
Yes, of course.
As you know, we have three different ways to write our UI when using UIKit. We can use Xib files, we can use Storyboards and we can use code. 
When we use the Xib files, Xcode will build those files into an intermediate representation called Nib. Then, when the view is loaded, the Nib file is read and the `awakeFromNib` function is invoked. We can override it to customise something programmatically.
When we use a code-first approach, as we do in our project, the `awakeFromNib` function won't be called, because we are not using the initializer that uses the `Nib` files. 

On top of that, if you look at the body of that function, it is just invoking `super.awakeFromNib`, so there is no point to have it explicitly written down, given that it is doing the same thing the parent will do.

Reply:
Perfect, Riccardo! thx for the detailed explanation :)